apps/evans/lib/color.ts

```diff
@@ -0,0 +1,36 @@
+import { MAROON_RGB, GOLD_RGB, BLUE_RGB, BLACK_RGB } from '@friends-library/theme';
+import { LANG } from '@/lib/env';
+
+export function bgLayer(
```

Comment:
this file/function i pulled over from the old gatsby dir, a helper for creating bg gradient layers that i used with gatsby-bg-image, and works well with our component as well.